src/drivers/mpu6000/mpu6000.h

```diff
@@ -204,12 +204,12 @@
 #define MPU6000_ACCEL_DEFAULT_DRIVER_FILTER_FREQ	30
 
 #define MPU6000_GYRO_DEFAULT_RANGE_G				8
-#define MPU6000_GYRO_DEFAULT_RATE					1000
+#define MPU6000_GYRO_DEFAULT_RATE					8000
```

Comment:
This is a bit above my head, but if the interrupt just fires faster than 280 Hz, the output rate. I'm OK. The FIFO buffer should handle all of the data and iterate through it integrating as it goes through the buffer, assuming the FIFO buffer is sufficiently large. Is there anything we need to do to configure the driver for that kind of sampling?